internal/host/host.go

```diff
@@ -51,6 +49,15 @@ var manualRebootStages = [...]models.HostStage{
 	models.HostStageDone,
 }
 
+var InstallationProgressTimeout = map[models.HostStage]time.Duration{
+	models.HostStageStartingInstallation: 10 * time.Minute,
+	models.HostStageWritingImageToDisk:   20 * time.Minute,
+	models.HostStageRebooting:            30 * time.Minute,
+	models.HostStageConfiguring:          30 * time.Minute,
+	models.HostStageWaitingForIgnition:   60 * time.Minute,
+	models.HostStageInstalling:           20 * time.Minute,
+}
```

Comment:
what about the other stages: HostStageJoined, HostStageWaitingForControlPlane, HostStageStartWaitingForControlPlane ?